openVirtualKeyboard.js

```diff
@@ -0,0 +1,35 @@
+
+/**
+ * Overview
+ *  - This is utility to open virtual keyboard in mobile.
+ * 
+ * Usage pattern:
+ *  - Import 
+ *    - import { openVirtualKeyboard } from '@dreamworld/web-util.js'
+ *  - Use
+ *    - openVirtualKeyboard();
+ *
+ */
+
+class OpenVirtualKeyboard {
```

Comment:
We don't use camel-case in the file name ever. It's always all small letters separated by dash.